### DISCUSSION TITLE
Smoke detector

### DISCUSSION THREAD

**Discussion by juntiedt2** [zwave-js/node-red-contrib-zwave-js#148] [Q&A] [answered]
Hi Marcus,

I received a WAKE-UP message from the smoke detector! 
This means, that the the controller number of 0 does not impact WAKE-UP message to controller 1.

**Comment by marcus-j-davies** [comment, marked as answer]
Hey! @juntiedt2 thats good news!

What this means:
The device has a quirk where it reports node 0 (or encodes it incorrectly in its raw message data), but behind the scene it is using Node 1.

its not uncommon at all for devices to have strange firmware oddities.

Feel free to use this discussion for anything you need. 👍
